week3/body_mass.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+
+'''
+Body Mass Index (BMI) is a measure of health on weight.
+It can be calculated by taking your weight in kilograms
+and dividing by the square of your height in meters.
+
+>>> bmi_calculator(65, 1.65)
+'Normal'
+'''
+
+def bmi_calculator(weight, height):
+
+    BMI = weight / height**2
+
+    TABLE = (
+        ((29.9), "Obese"),
+        ((24.9), "Overweight"),
+        ((18.5), "Normal"),
+        ((0), "Underweight"),
+        )
+
+    return [ TABLE[TABLE.index(data)][1] for data in TABLE if BMI > data[0] ][0]
```

Comment:
this is not easily readable... what about a simple if else statement (think of the developers who are going to read your code, which could be you in 6 months time as well)?
A classic: "Always code as if the person who ends up maintaining your code is a violent psychopath who knows where you live." http://blog.codinghorror.com/coding-for-violent-psychopaths/